lib/payload.json

```
@@ -128,7 +128,11 @@
           "type": "object",
           "properties": {
             "speed": {
-              "description": "Wind speed (m/s)",
+              "description": "Instantaneous or average wind speed (m/s)",
```

I prefer to avoid documenting/promoting the use of average here until we figure out a way to introduce grouping and stats.
```suggestion
              "description": "Wind speed (m/s)",
```